api/app/src/Entity/CourtOrderDeputy.php

```diff
@@ -0,0 +1,76 @@
+<?php
+
+namespace App\Entity;
+
+use Doctrine\ORM\Mapping as ORM;
+
+/**
+ * Link table between court orders and deputies.
+ *
+ * @ORM\Table(name="court_order_deputy")
+ *
+ * @ORM\Entity()
+ *
+ * @ORM\HasLifecycleCallbacks()
+ */
+class CourtOrderDeputy
+{
+    /**
+     * @ORM\Id
+     *
+     * @ORM\ManyToOne(targetEntity="App\Entity\CourtOrder", inversedBy="courtOrderDeputyRelationships", cascade={"persist"})
+     *
+     * @ORM\JoinColumn(name="court_order_id", referencedColumnName="id")
+     */
+    private CourtOrder $courtOrder;
```

Comment:
Think the 2 ids should be non nullable as they would always both be populated if the row exists right?

Reply:
that is if nullable if false by default which i think it is

Reply:
Good point, I've updated the entity class